interfaces/Portalicious/src/app/pages/project/project-team/add-user-form/add-user-form.component.ts

```diff
@@ -50,23 +52,49 @@ type AddUserToTeamFormGroup =
     ReactiveFormsModule,
   ],
 })
-export class AddUserButtonComponent {
+export class AddUserFormComponent {
   projectId = input.required<number>();
-  enableScope = input.required<boolean | undefined>();
+  enableScope = input.required<boolean>();
+  formVisible = model.required<boolean>();
```

Comment:
Aha! `model` was the trick I was looking for... I fiddled way to long with `input`/`signal` etc.. 🤦